app/src/main/java/eu/kanade/tachiyomi/data/download/DownloadProvider.kt

```diff
@@ -165,24 +196,30 @@ class DownloadProvider(
     }
 
     fun isChapterDirNameChanged(oldChapter: Chapter, newChapter: Chapter): Boolean {
```

Comment:
Now that I think about it this function doesn't really account for name schema change

Reply:
Was this accounted for?

Reply:
You said it could be a separate PR, so changes to it aren't included here.